### PR TITLE
Allow passing params to RocketfuelApi::Service#get_all

### PR DESCRIPTION
**Pull request by guigs**
To import RocketFuel Line Items, it is required to pass `campaign_id` as param to `get_all`.

### CODE REVIEW

File: spec/importeur/data_sources/rocketfuel_spec.rb [hunk 1]
```diff
@@ -9,12 +9,13 @@
...
 
   before do
-    expect(rocketfuel_service).to receive(:get_all).and_return([resource_resource])
+    allow(rocketfuel_service).to receive(:get_all).and_return([resource_resource])
   end
 
   describe '#items' do
     it 'returns items' do
       expect(data_source.items).to eq([resource_resource])
+      expect(rocketfuel_service).to have_received(:get_all).with({})
```
Comment (RKushnir) [inline spec/importeur/data_sources/rocketfuel_spec.rb]: By the name `get_all` I assume this method doesn't have side-effects, so we don't need to "expect" it. What we care about is that `#items` returns correct value depending on the params. And we can test it:
```ruby
before do
  allow(rocketfuel_service).to receive(:get_all).with({}).and_return([resource_resource])
  allow(rocketfuel_service).to receive(:get_all).with(params).and_return(specific_items)
end

it 'returns items for default params' do
  expect(data_source.items).to eq([resource_resource])
end

context 'with params' do
  …
  it 'returns items for custom params' do
   expect(data_source.items).to eq(specific_items)
  end
end
```

Comment (RKushnir) [reply spec/importeur/data_sources/rocketfuel_spec.rb]: The last row, for outgoing messages, we only need to "expect" command messages, which change something, but not queries, which just return data.

Comment (guigs) [reply spec/importeur/data_sources/rocketfuel_spec.rb]: Interesting. I had to look for the presentation to understand it though :wink:

Comment (RKushnir) [reply spec/importeur/data_sources/rocketfuel_spec.rb]: Yeah, I suspected the image itself might not be too descriptive.

Comment (RKushnir) [reply spec/importeur/data_sources/rocketfuel_spec.rb]: There's also a book from the author, http://www.poodr.com/. One of those actually good books :) And quite short, also.

**Pull request merged**